### PR TITLE
Adding GitHub Marketplace topics

### PR DESCRIPTION
**Pull request by chobberoni**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Unaffiliated with the project (not self-promoting as e.g. a maintainer, creator, contractor or employee)

************EDITING AN EXISTING TOPIC OR COLLECTION************

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

Helping out the internal topics team.

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:

We need more things to be globally indexed.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

cc: @brandonrosage @andymckay

### CODE REVIEW

File: topics/localization/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: localization, localisation
+display_name: Localisation
```
Comment (talsafran) [inline topics/localization/index.md]: Super minor detail: the directory name here is US-centric "locali**z**ation", but the display name, topic, and description have the UK-centric "locali**s**ation". Should we make them consistent? 

I probably have a slight preference for the 💤 , fwiw.

Comment (talsafran) [reply topics/localization/index.md]: Ooooo looks like [CI caught this](https://travis-ci.org/github/explore/builds/391481676?utm_source=github_status&utm_medium=notification):

```
  8) Failure:
topics::localisation topic#test_0003_does not add an alias that's already in use [/home/travis/build/github/explore/test/topics_test.rb:32]:
localization,  localisation are already aliased to localization, please remove from either 'localisation' or 'localization'.
Expected ["localization", " localisation"] to be empty.
```

There are a bunch of other failures 🤔 . Mind taking a look?

File: topics/code-review/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: code-review, revew, gerrit
+display_name: Code Review
+short_description: Ensure your code meets quality standards and ship with confidence.
+topic: code-review
+
+---
+Ensure your code meets quality standards and ship with confidence.
```
Comment (leereilly) [inline topics/code-review/index.md]: I'd make this more general and opt for something like [Wikipedia's description](https://en.wikipedia.org/wiki/Code_review) (not verbatim):

> Code review is systematic examination (sometimes referred to as peer review) of computer source code. It is intended to find mistakes overlooked in software development, improving the overall quality of software. Reviews are done in various forms such as pair programming, informal walkthroughs, and formal inspections.[1]

File: topics/code-quality/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: code-quality, quality
+display_name: Code Quality
```
Comment (brandonrosage) [inline topics/code-quality/index.md]: Should be sentence case: "Code quality"

File: topics/code-review/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: code-review, revew, gerrit
+display_name: Code Review
```
Comment (brandonrosage) [inline topics/code-review/index.md]: Should be sentence case: "Code review"

File: topics/continuous-integration/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: continuous-integration, ci
+display_name: Continuous Integration
```
Comment (brandonrosage) [inline topics/continuous-integration/index.md]: Should be sentence case: "Continuous integration"

File: topics/dependency-management/index.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: dependencies, dependency-manager
+display_name: Dependency Management
```
Comment (brandonrosage) [inline topics/dependency-management/index.md]: Should be sentence case: "Dependency management"

File: topics/localization/index.md (hunk 1 above)
Comment (chobberoni) [reply topics/localization/index.md]: solved in my new PR -- coming soon

File: topics/mobile/index.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: mobile
+display_name: Mobile
+short_description: Improve your workflow for the small screen.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/mobile/index.md]: Added something general in the new PR

File: topics/continuous-integration/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/continuous-integration/index.md]: Removed.

File: topics/ci/index.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: continuous-integration, ci
+display_name: CI
+short_description: Automatically build and test your code as you push it to GitHub, preventing bugs from being deployed to production.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/ci/index.md]: updated and made it more general.

File: topics/code-review/index.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: code-review, revew, gerrit
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/code-review/index.md]: fixed!

File: topics/project-management/index.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: project-management
+display_name: Project Management
+short_description: Organize, manage, and track your project with tools that build on top of issues and pull requests.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/project-management/index.md]: changed

File: topics/support/index.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: publishing
+display_name: Publishing
+short_description: Get your site ready for production so you can get the word out.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/support/index.md]: updated

File: topics/code-review/index.md (hunk 2 above)
Comment (chobberoni) [reply topics/code-review/index.md]: updated

**Comment by chobberoni** [comment]
@brandonrosage @andymckay Just pushed a ton of updates upstream. Hopefully this meets our requirements and passes CI

**Comment by chobberoni** [comment]
@leereilly @brandonrosage Can I get a re-review here now that things have passed CI 😃?

Please and thank you!

**Comment by chobberoni** [comment]
Friendly bump for @leereilly as this is slated to ship internally on Tuesday.

File: topics/mobile/index.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: mobile-apps
+display_name: Mobile
+short_description: Tools to improve your workflow for the small screen.
```
Comment (leereilly) [inline topics/mobile/index.md]: I think this short description is still too focused on Marketplace / one particular use case. If we search for repos tagged [`#mobile`](https://github.com/topics/mobile?o=desc&s=updated) we'll see full blown apps, games, operating systems, databases, etc. They would quite fit this description...

screen_shot_2018-06-23_at_11_01_52_am

Maybe make the description overly generic, remove it, or make a few more granular options e.g. `#mobile-app-development.md`, `mobile-framework.md`?

File: topics/localisation/index.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Localisation
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/localisation/index.md]: removed

File: topics/localization/index.md [hunk 14]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Localization
+short_description: Extend your software's reach. Localize and translate continuously from GitHub.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/localization/index.md]: Short description changed. No i18n added.

File: topics/project-management/index.md [hunk 15]
```diff
@@ -0,0 +1,8 @@
+---
+display_name: Project management
+short_description: Organize, manage, and track your project with tools that build on top of issues and pull requests.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/project-management/index.md]: changed to match the long description.

File: topics/publishing/index.md [hunk 16]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Publishing
+short_description: Get your site ready for production so you can get the word out.
```
[the remark this answers is not in the document]
Comment (chobberoni) [reply topics/publishing/index.md]: updated

**Comment by chobberoni** [comment]
@leereilly Can we get one last look over as the feature that is dependent upon this PR goes out today?

cc: @andymckay

**Pull request merged**